sky/core.py

```diff
@@ -757,3 +757,21 @@ def storage_delete(name: str) -> None:
                                     source=handle.source,
                                     sync_on_reconstruction=False)
         store_object.delete()
+
+
+def storage_create(name: Optional[str] = None,
+                   source: Optional[str] = None,
+                   stores: Tuple[str] = None):
+    """"Create a storage.
+
+    """
+    storage = data.Storage(name, source)
+
+    stores = [] if stores is None else stores
+    stores = [store_type.upper() for store_type in stores]
```

Comment:
Move this check into `storage.py`?

Reply:
The add_store method takes in a StoreType in `storage.py`. Should I make a separate function that calls add_store that takes in a list of strings as a parameter?

Reply:
Handled alongside concurrency